chains/evm/contracts/onRamp/CCVProxy.sol

```diff
@@ -372,9 +383,17 @@ contract CCVProxy is IEVM2AnyOnRampClient, ITypeAndVersion, Ownable2StepMsgSende
       resolvedArgs.executor = destChainConfig.defaultExecutor;
     }
 
-    // Add the required CCV, if set.
+    // Enforce chain-level security requirements: some destination chains mandate specific CCVs for all messages.
```

Comment:
"some destination chains mandate specific CCVs for all messages" isn't really saying anything here. Let's prefer concise comments